instrumentation/opentelemetry-annotations-1.0/README.md

```diff
@@ -0,0 +1,6 @@
+# Settings for the OpenTelemetry Annotations integration
+
+| Environment variable 	| Type 	| Default 	| Description 	|
+|-----------------	|------	|---------	|-------------	|
+| `TRACE_ANNOTATED_METHODS_EXCLUDE_CONFIG` | String |  | All methods to be excluded from auto-instrumentation by annotation-based advices. |
```

Comment:
`TRACE_ANNOTATED_METHODS_EXCLUDE_CONFIG` is not the env var/system prop name, it's the constant name. The real system property name is `otel.instrumentation.external-annotations.exclude-methods`